### PR TITLE
Foot follows

### PR DESCRIPTION
**Pull request by wheresrhys**
@ironsidevsquincy

### CODE REVIEW

**Comment by wheresrhys** [comment]
image

File: client/components/more-on/_main.scss [hunk 1]
```diff
@@ -1,105 +1,112 @@
...
-	padding: 5px;
-	font-size: 17px;
-	font-weight: 200;
-	line-height: 1em;
+	margin: 0;
+	padding: 5px 0;
+	@include nTypeAlpha(4);
 	box-sizing: border-box;
 
+	a {
```
Comment (ironsidevsquincy) [inline client/components/more-on/_main.scss]: no class name available?

**Comment by matthew-andrews** [comment]
looks aweseom :+1:

File: client/components/more-on/_main.scss [hunk 2]
```diff
@@ -1,105 +1,112 @@
...
+	.more-on--people .more-on__title:before {
+		@include nextIcon(profile, oColorsGetPaletteColor(cold-3), 30);
+	}
+
+	.more-on--regions .more-on__title:before {
+		@include nextIcon(section-world, oColorsGetPaletteColor(cold-3), 30);
+	}
+
+	.more-on--topics .more-on__title:before {
+		@include nextIcon(section-special-report, oColorsGetPaletteColor(cold-3), 30);
```
Comment (ironsidevsquincy) [inline client/components/more-on/_main.scss]: seeing all these, I wonder if we should have the ability to use `nextIcon` which just adds the unique styles, as we're repeating things like `display: inline-block` etc for each one of these

Comment (wheresrhys) [inline client/components/more-on/_main.scss]: css-clean was sorting this for us, but not sure any more. yes good idea

**Comment by ironsidevsquincy** [comment]
:dancers: :dancers:  :dancers: :dancers:

**Pull request merged**